### PR TITLE
airbnb eslint rules applied

### PR DESCRIPTION
**Pull request by adevnadia**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -265,14 +265,14 @@ function unregisterNode (host, node, func) {
   }
 }
 
-function addNodeToNode (host, node, insertBefore) {
-  registerNode(host, node, insertBefore, function (eachNode) {
+function addNodeToNode(host, node, insertBefore) {
+  registerNode(host, node, insertBefore, (eachNode) => {
```
Comment (bengummer) [inline src/index.js]: parens around eachNode isn't technically required here - is this is the style we want? just double checking :)

File: src/index.js [hunk 2]
```diff
@@ -441,54 +444,54 @@ function appendChildOrInsertBefore (host, newNode, refNode) {
...
     },
-    set (value) {
+    set(value) {
       this.______slotChangeListeners = value;
-    }
+    },
   },
   ____triggerSlotChangeEvent: {
-    value: debounce(function () {
+    value: debounce(function callback() {
```
Comment (bengummer) [inline src/index.js:471]: this could be fat arrow `() => { ... }` syntax

File: src/index.js [hunk 3]
```diff
@@ -638,125 +649,121 @@ const members = {
...
+    },
   },
   outerHTML: {
-    get () {
+    get() {
       const name = this.tagName.toLowerCase();
-      const attributes = Array.prototype.slice.call(this.attributes).map(function (attr) {
-        return ` ${attr.name}${attr.value ? `="${attr.value}"` : ''}`;
-      }).join('');
+      const attributes = Array.prototype.slice.call(this.attributes).map((attr) => (` ${attr.name}${attr.value ? `="${attr.value}"` : ''}`)).join(''); // eslint-disable-line max-len
```
Comment (bengummer) [inline src/index.js]: i think the old `attributes` assignment was much more readable. would be good to split the map function onto a new line, not sure having it all on the same line is worth it.

File: src/index.js (hunk 1 above)
Comment (adevnadia) [inline src/index.js]: Either ways are fine with airbnb

File: src/index.js [hunk 4]
```diff
@@ -638,125 +649,121 @@ const members = {
...
+    },
   },
   parentElement: {
-    get () {
-      return findClosest(this.parentNode, function (node) {
-        return node.nodeType === 1;
-      });
-    }
+    get() {
+      return findClosest(this.parentNode, (node) => node.nodeType === 1);
```
Comment (bengummer) [inline src/index.js:727]: 👍

File: src/index.js [hunk 5]
```diff
@@ -836,29 +843,30 @@ if (!('attachShadow' in document.createElement('div'))) {
       const nativeDescriptor = getPropertyDescriptor(elementProto, memberName);
       const nativeTextDescriptor = getPropertyDescriptor(textProto, memberName);
       const nativeCommDescriptor = getPropertyDescriptor(commProto, memberName);
-      const shouldOverrideInTextNode = (memberName in textNode && doNotOverridePropertiesInTextNodes.indexOf(memberName) === -1) || ~defineInTextNodes.indexOf(memberName);
-      const shouldOverrideInCommentNode = (memberName in commNode && doNotOverridePropertiesInCommNodes.indexOf(memberName) === -1) || ~defineInCommNodes.indexOf(memberName);
+      const shouldOverrideInTextNode = (memberName in textNode && doNotOverridePropertiesInTextNodes.indexOf(memberName) === -1) || ~defineInTextNodes.indexOf(memberName); // eslint-disable-line max-len
+      const shouldOverrideInCommentNode = (memberName in commNode && doNotOverridePropertiesInCommNodes.indexOf(memberName) === -1) || ~defineInCommNodes.indexOf(memberName); // eslint-disable-line max-len
+      const nativeMemberName = '__' + memberName;
```
Comment (bengummer) [inline src/index.js]: 👍

File: src/util/get-property-descriptor.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ const div = document.createElement('div');
 function getPrototype(obj, key) {
   let descriptor;
 
-  while (obj && !(descriptor = Object.getOwnPropertyDescriptor(obj, key))) {
+  while (obj && !(descriptor = Object.getOwnPropertyDescriptor(obj, key))) { // eslint-disable-line max-len, no-cond-assign
```
Comment (bengummer) [inline src/util/get-property-descriptor.js]: it seems like the eslint max-line length rule is disabled a few times, is it worth us overriding this rule with a higher value?

**Pull request merged**